packages/driver/src/cypress/script_utils.ts

```diff
@@ -25,11 +24,15 @@ const extractSourceMap = ([script, contents]) => {
 }
 
 const evalScripts = (specWindow, scripts: any = []) => {
-  _.each(scripts, ([script, contents]) => {
-    specWindow.eval(`${contents}\n//# sourceURL=${script.fullyQualifiedUrl}`)
-  })
+  return Bluebird.each(scripts, (_script: any) => {
+    const [script, contents] = _script
+
+    if (script.load) {
+      return script.load()
+    }
 
-  return null
+    return specWindow.eval(`${contents}\n//# sourceURL=${script.fullyQualifiedUrl}`)
+  })
```

Comment:
Small cleanup we can do below in `runScripts`, the `if (scripts.length && typeof scripts[0] === 'function') {` is never valid now since we are sending up the spec payload now for CT, so it can be simplified to:

```ts
export default {
  runScripts: runScriptsFromUrls
}
```